src/skin/legacy/legacyskinparser.cpp

```diff
@@ -1300,10 +1300,13 @@ QWidget* LegacySkinParser::parseSpinny(const QDomElement& node) {
 }
 
 QWidget* LegacySkinParser::parseVuMeter(const QDomElement& node) {
-    if (!CmdlineArgs::Instance().getUseVuMeterGL()) {
+    auto* pWaveformWidgetFactory = WaveformWidgetFactory::instance();
+    if (!CmdlineArgs::Instance().getUseVuMeterGL() |
+            (!pWaveformWidgetFactory->isOpenGlAvailable() &&
```

Comment:
typo which results in different semantics... Would probably have been caught by `-Wconversion`
```suggestion
    if (!CmdlineArgs::Instance().getUseVuMeterGL() ||
            (!pWaveformWidgetFactory->isOpenGlAvailable() &&
```

Reply:
I will create a PR to fix this!

Reply:
Thank you.

Reply:
Here it is: #11292